index.js

```diff
@@ -359,6 +363,83 @@
       [a, b, a],
       function(x, y) { return x; });
 
+  //# A :: (a -> b) -> a -> b
+  //.
+  //. The A combinator. Takes a function and a value and returns the result of
+  //. applying the function to the value. Equivalent to Haskell's `($)` function.
+  //.
+  //. ```javascript
+  //. > S.A(R.add(1), 1)
+  //. 2
+  //.
+  //. > R.map(S.flip(S.A, 3), [R.add(2), R.multiply(2)])
+  //. [5, 6]
+  //. ```
+  S.A =
+  def('A',
+      {},
+      [$.Function, a, b],
+      function(f, x) { return f(x); });
+
+  //# C :: (a -> b -> c) -> b -> a -> c
+  //.
+  //. The C combinator. Takes a curried binary function and two values and
+  //. returns the result of applying the function to the values in reverse.
+  //. Equivalent to Haskell's `flip` function.
+  //.
+  //. This function is very similar to `S.flip`, except that its first argument
+  //. must be curried. This also allows it to work with manually curried
+  //. functions.
+  //.
+  //. ```javascript
+  //. > R.gt(1, 2)
+  //. false
+  //.
+  //. > S.C(R.gt, 1, 2)
+  //. true
```

Comment:
Ah, I see now why you chose this example. To show the problem with `R.gt` we'd need to partially apply the function in an expression such as:

```javascript
> R.map(S.C(R.gt, 0), [-1, -2, 3, -4, 4, 2])
[3, 4, 2]
```

I like the opening example to be trivial, even if it doesn't demonstrate realistic usage. I'd be happy with something like the above as the second example, so feel free to replace the `'KA-POW!'` one. :)